src/common-tests.js

```diff
@@ -522,20 +535,23 @@ function common (app, errors, serviceName = 'people', idProp = 'id') {
         }).then(() => service.remove(null, params));
       });
 
-      it('patches multiple even if query changed', () => {
+      it('patches multiple, returns correct items', () => {
         const service = app.service(serviceName);
```

Comment:
The current (buggy) implementations make just another `find` with the changed query which is not correct. What they really need to do is `find` everything before making the database update and then do another find with the list of all ids from the previous find. This tests covers that.